RAW/ViewController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  ViewController.swift
+//  RAW
+//
+//  Created by Bharat malhotra on 13/07/18.
+//  Copyright © 2018 Bharat malhotra. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+
```

Comment:
can we put the tableview delegate and datasource methods in this class's separate extension?

Reply:
Yes